client/src/components/SearchResult.js

```diff
@@ -165,32 +180,35 @@ export const RequestRequirements = ({ resource }) => {
 
   const requirements = []
   const MTA = 'Material Transfer Agreement'
+  if (resource.needs_abstract) requirements.push('Abstract')
   if (resource.needs_irb) requirements.push('IRB')
   if (resource.needs_mta) requirements.push(MTA)
-  if (resource.needs_mta) requirements.push('Abstract')
-  // if (resource.needs_shipping) requirements.push('Shipping Information')
+  if (resource.needs_shipping_info) requirements.push('Shipping Information')
 
   return (
-    <SearchResultDetail title="Request Requirements">
+    <SearchResultDetail
+      title="Request Requirements"
+      margin={{ top: 'small' }}
+      direction="row"
+    >
       {requirements.length === 0 && 'Not Available'}
       {requirements.length > 0 && (
-        <Paragraph margin={{ vertical: 'small' }}>
-          {requirements.map((req, i) => (
-            <React.Fragment key={req}>
-              {i !== 0 && `, `}
-              {req === MTA ? (
-                <Anchor
-                  href={resource.mta_s3_url}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  label={req}
-                />
-              ) : (
-                <Text>{req}</Text>
-              )}
-            </React.Fragment>
-          ))}
-        </Paragraph>
+        <span>
+          {requirements.map((req, i) => [
+            i !== 0 && ', ',
+            req === MTA ? (
+              <Anchor
+                key={req}
+                href={resource.mta_s3_url}
```

Comment:
Note that this field has gone away.

Reply:
good catch I can update after the attachments stuff is in the books

Reply:
added a note too

Reply:
In which books?

Reply:
I think we would need the attachment in the material serializer in order to show the link to it?